src/app/components/data-entry-grid/index.html

```diff
@@ -0,0 +1,232 @@
+          </p>
+        </sky-demo-page-property>
+
+        <sky-demo-page-property
+          propertyName="SkyCellType.Number"
+          isOptional="true"
+        >
+          <p>
+            In edit mode, a cell in a number column will render as an HTML number <sky-code>input</sky-code>. Only numbers can be entered.
+          </p>
+
+          <p>
+            In read-only mode, the cell will render as the number value.
+          </p>
+        </sky-demo-page-property>
+
+        <sky-demo-page-property
+          propertyName="SkyCellType.RowSelector"
+          isOptional="true"
+        >
+          In edit or read-only mode, a cell in a row selector column will render as a <a routerLink="../checkbox">SKY UX checkbox component.</a> It allows the user to select multiple rows, and adds a highlight to selected rows. The <a href="https://www.ag-grid.com/javascript-grid-row-node/">ag-Grid <sky-code>rowNode</sky-code></a> will be updated to reflect the selected state.
+        </sky-demo-page-property>
+
+
+        <sky-demo-page-property
+          propertyName="SkyCellType.Text"
+          isOptional="true"
+        >
+          <p>
+            In edit mode, a cell in a text column will render as an HTML text <sky-code>input</sky-code>. Any characters can be entered.
+          </p>
+
+          <p>
+            In read-only mode, the cell will render as the string value.
+          </p>
+        </sky-demo-page-property>
+
+        <p>
+          Any <a routerLink="..">SKY UX component</a> can be made into a <a href="https://www.ag-grid.com/javascript-grid-cell-editor/">cell editor</a> or <a href="https://www.ag-grid.com/javascript-grid-cell-rendering-components/">cell renderer</a> component. If you would like to use a component that does not have a column definition yet, please consider <a routerLink="../../contribute/contribution-process">contributing it</a> to the SKY UX ag-Grid module, or <a routerLink="../../contribute/contribution-process/file-issue">file an issue</a> in the <a href="https://github.com/blackbaud/skyux-ag-grid">@skyux/ag-grid repo.</a>
+        </p>
+      </sky-demo-page-properties>
+
+      <h2>Passing cell editor parameters</h2>
+      <p>
+        You can pass inputs to the SKY UX components used in cell editors via the <a href="https://www.ag-grid.com/javascript-grid-column-properties/">column definition's <sky-code>cellEditorParams</sky-code> property.</a> The component properties are nested under <sky-code>skyComponentProperties</sky-code>. All component properties should be supported, and you can see the defined types by importing <sky-code>SkyCellEditorDatepickerProperties</sky-code> or <sky-code>SkyCellEditorAutocompleteProperties</sky-code>. <sky-code>cellEditorParams</sky-code> can also be a function that returns a param object for dynamic editing params. See the demo for examples of using an object or a function.
```

Comment:
We could probably add a link to the demo or demo code, whichever will be more helpful to the user.

Reply:
Currently, the demo appears right after this section, and the code appears right after that. So for now, I think we are good without the link. And we can always add the link later if we decide it would be helpful.